buddyboss/buddyboss-platform
# PROD-918 #1437 - Fix PHP Error: Display name format conflict

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #1437 

### How to test the changes in this Pull Request:

1. Go to this page https://demos.buddyboss.com/online-communities/wp-admin/admin.php?page=bp-settings&tab=bp-xprofile

2. Choose these settings https://gyazo.com/383787e084e20e1084a56aa8532821e1 and Save changes.

3.. Then go to the Checkout settings https://demos.buddyboss.com/online-communities/wp-admin/admin.php?page=wc-settings&tab=account and choose this setting: https://gyazo.com/6108f95f5d88931ad5269310e6b2363e

4. Then logout from the website. And create an order as a new client: https://gyazo.com/316b015fe7d51b4fb3b0a6741b6de05d

5. Now go to the profile page of that user and see error. : https://prnt.sc/ua685n

So the main issue with the "Display name publicly as" filed. It is showing the last name on the page title.

### Other information:
